js/application/init.js

```diff
@@ -242,7 +247,12 @@ define([
           document.title = this.config.title;
 
           // Launch the app specifics
-          this.main.start(this.config, dom.byId("viewDiv"), view, webscene, null, null);
+          this.main.start({
+            config: this.config,
+            viewDiv: dom.byId("viewDiv"),
+            view: view,
+            mapOrScene: webscene
```

Comment:
@MikeTschudi can we just get this renamed to `scene:...` or `map:...`? That way users know which it is.